gift-certificates-app/dao/src/test/java/com/epam/esm/dao/impl/UserDAOImplTest.java

```diff
@@ -31,29 +29,16 @@
 
     @Autowired
     private UserDAO userDAO;
-    private static User userExpected;
-    private static User userExists;
 
     @BeforeAll
     public static void initUser() {
-        userExpected = new User();
-        userExpected.setLogin(NEW_USER);
+        User userExpected = new User();
+        userExpected.setUsername(NEW_USER);
+        userExpected.setPassword(NEW_USER);
 
-        userExists = new User();
-        userExists.setLogin(USER_EXISTS);
-    }
-
-    @Test
-    @Transactional
-    void add() {
```

Comment:
Looks like User add(User user) and Optional<User> get(String username) not covered

Reply:
done